cherab/tools/inversions/opencl/sart_opencl.py

```diff
@@ -0,0 +1,256 @@
+# -*- coding: utf-8 -*-
+# Created by Vladislav Neverov (NRC "Kurchatov Institute") for CHERAB Spectroscopy Modelling Framework
+from __future__ import print_function
+import os
+import numpy as np
+import pyopencl as cl
```

Comment:
Similarly here, can you make this a try/except import only called when this class is instantiated? Keep the reference to the module on the class maybe.

Reply:
Done.